docs/docs/04-standard-library/std/array.md

```diff
@@ -351,16 +351,16 @@ the index to remove the value at.
 ##### `push` <a name="push" id="@winglang/sdk.std.MutArray.push"></a>
 
 ```wing
-push(...values: Array<<T>>): void
+push(value: <T>): void
```

Comment:
Regression?

Reply:
fixed by PR mutation, thanks for noticing!